x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -310,63 +287,71 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
             onMouseLeave={onSwimlaneLeaveHandler}
             data-test-subj="mlAnomalyExplorerSwimlaneOverall"
           >
-            {showOverallSwimlane && (
-              <SwimlaneContainer
-                filterActive={filterActive}
-                maskAll={maskAll}
-                timeBuckets={timeBuckets}
-                swimlaneCellClick={swimlaneCellClick}
-                swimlaneData={overallSwimlaneData as OverallSwimlaneData}
-                swimlaneType={'overall'}
-                selection={selectedCells}
-                swimlaneRenderDoneListener={swimlaneRenderDoneListener}
-                onResize={(width) => explorerService.setSwimlaneContainerWidth(width)}
-              />
-            )}
+            <SwimlaneContainer
```

Comment:
When there are no results, I think the styling of the two 'no data' messages should be consistent, in terms of alignment and font size. The overall data message should have the same styling as the view by swimlane I think:

![image](https://user-images.githubusercontent.com/7405507/86233088-cef1f380-bb8c-11ea-930c-852f891e58fd.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/70063/commits/04bf4ca21a113892b23c4d5391f84833beb71b25